svc/api/admin/src/route/mod.rs

```diff
@@ -20,15 +22,38 @@ pub async fn handle(
 
 define_router! {
 	routes: {
+		"login": {
+			POST: login::login(
+				body: models::AdminLoginRequest,
+			),
+		},
+
 		"cluster" / "server_ips": {
 			GET: cluster::server_ips(
 				query: cluster::ServerIpsQuery,
 			),
 		},
 
-		"login": {
-			POST: login::login(
-				body: models::AdminLoginRequest,
+		"cluster": {
+			POST: cluster::create(
+				body: models::AdminClusterCreateRequest,
+			),
+			DELETE: cluster::delete(),
+		},
+
+		"cluster" / Uuid / "datacenter": {
+			POST: cluster::datacenter::create(
+				body: models::AdminClusterDatacenterCreateRequest,
+			),
+		},
+
+		"cluster" / Uuid / "datacenter" / Uuid : {
```

Comment:
dc should take name id as path parameter

Reply:
This wouldn't match our existing API structure, i think we should resolve name ids in bolt and use the uuid in the path.

Reply:
ah yeah good call @MasterPtato